pandas-stubs/core/indexing.pyi

```diff
@@ -58,17 +58,21 @@ class _LocIndexerFrame:
     @overload
     def __getitem__(self, idx: _StrLike,) -> Series: ...
     @overload
-    def __getitem__(self, idx: Tuple[_StrLike, _StrLike],) -> float: ...
+    def __getitem__(self, idx: List[_StrLike],) -> DataFrame: ...
     @overload
     def __getitem__(
         self, idx: Tuple[Union[_MaskType, List[str]], Union[_MaskType, List[str]]],
     ) -> DataFrame: ...
+    @overload
+    def __getitem__(self, idx: Tuple[_StrLike, _StrLike],) -> float: ...
     # set item
     @overload
     def __setitem__(self, idx: _MaskType, value: Union[float, _np.ndarray, DataFrame],) -> None: ...
     @overload
     def __setitem__(self, idx: _StrLike, value: Union[float, Series, _np.ndarray],) -> None: ...
     @overload
+    def __setitem__(self, idx: Tuple[_IndexType, str], value: Union[_IndexType, float]) -> None: ...
```

Comment:
I don't really like that this `float` is here

Reply:
I think it's more correct than the `_IndexType` to be honest.